docs/dev/config/lookout/config.yaml

```diff
@@ -0,0 +1,44 @@
+grpcPort: 50059
+httpPort: 8089
+metricsPort: 9009
+
+grpc:
+  keepaliveParams:
+    maxConnectionIdle: 5m
+    time: 2h
+    timeout: 20s
+  keepaliveEnforcementPolicy:
+    minTime: 5m
+    permitWithoutStream: false
+
+uiConfig:
+  armadaApiBaseUrl: "http://localhost:8080"
```

Comment:
I don't think this is correct. Shouldn't we use container name for armada server or lookout?

Reply:
I think this works somehow!